keps/sig-auth/20190408-node-scoped-daemonset.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Node-Scoped DaemonSet
+authors:
+  - "@haiyanmeng"
+owning-sig: sig-auth
+participating-sigs:
+  - sig-auth
```

Comment:
nit: I think the owning-sig is implied here. I'd add sig-node though.

Reply:
Fixed it! Thanks!